test/lit/passes/local-cse_all-features.wast

```diff
@@ -87,7 +129,7 @@
     ;; Repeated loads from a struct can be optimized.
     ;;
     ;; Note that these struct.gets cannot trap as the reference is non-nullable,
-    ;; so there are no side effects here, and we can optimize.
+    ;; so this is "easier" than the previous testcase.
```

Comment:
Isn't it the same given that we ignore traps now? "Easier" sounds like somehow this one does something different..

Reply:
I'll clarify.